cdap-data-fabric/pom.xml

```diff
@@ -172,6 +172,12 @@
       <artifactId>hbase-testing-util</artifactId>
       <scope>test</scope>
     </dependency>
+
+    <!--TODO: this dependency will need to be moved to test scope when we use the jdbc driver to create connection-->
```

Comment:
Why it cannot be in test scope now? It seems like you only use it in test??

Reply:
This will be needed in the next store migration pr, if we want the store to use the embedded postgres sql, we will have to have this not in test scope. I can put it in test scope in this pr

Reply:
I don't understand why it needs to be in non-test scope when Store starts using `SqlStructuredTable`?

Reply:
If we want to run on sandbox to verify the sql implementation, we will need to have this dependency non-test. If we just want to migrate the store so that we make sure the unit test pass, we can have it in test scope. But eventually this should go to test scope since we will have the actual Postgres driver to provide the connection.